template/private-npm-package/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "<%- fullModuleName %>",
+  "description": "<%- description %>",
+  "version": "0.0.0-semantically-released",
+  "main": "lib-commonjs",
+  "module": "lib-es2015",
+  "typings": "lib-types",
+  "files": [
+    "lib"
+  ],
+  "sideEffects": false,
+  "license": "UNLICENSED",
+  "scripts": {
+    "build": "skuba build",
+    "commit": "skuba commit",
+    "format": "skuba format",
+    "format:check": "skuba format check",
+    "lint": "skuba lint",
+    "prerelease": "skuba build",
+    "release": "skuba release",
+    "test": "echo \"No tests for now\""
```

Comment:
Could we create a vanity test like we have in the `greeter` template? It makes it a bit easier to get started for the user.